projects/fhi-angular-components/README.md

```diff
@@ -6,6 +6,8 @@ Contains frontend Angular components used by FHI, based on [Bootstrap widgets (@
 
 | FHI AngularComponents | Fhi Style | Bootstrap | NgSelect | NgBootstrap | Angular | Node/NPM |
 | --------------------- | --------- | --------- | -------- | ----------- | ------- | -------- |
+| 0.2.0                 | 4.0.0     | 5         | 9        | 13          | 14      | 16/8 *   |
```

Comment:
We should probably either write Fhi Style 4 or Fhi Style ^4.0.0 to be precise here... I think just 4 is best since thats what we do for Bootstrap etc.